proto/contimage/contimage.proto

```diff
@@ -0,0 +1,40 @@
+syntax = "proto3";
+
+option go_package = "github.com/DataDog/agent-payload/v5/contimage";
+
+package datadog.contimage;
+
+// ContainerImagePayload represents the main container image payload
+message ContainerImagePayload {
+    string                  version = 1;  // version represents the ContainerImagePayload message version
+    string                  host    = 2;  // host contains the hostname
+    repeated ContainerImage images  = 3;  // Images metadata
+}
+
+// ContainerImage contains the details of a container image
+message ContainerImage {
```

Comment:
Out of the scope of my understanding on the details of this feature and not a blocker but would adding [content trust fields](https://www.howtogeek.com/devops/how-to-sign-your-docker-images-to-increase-trust/#:~:text=Signing%20Images,with%20the%20correct%20registry%20URL.&text=This%20command%20will%20sign%20the,straight%20up%20to%20the%20registry) around the image tags be valuable here since it can be used to identify modified images/layers?